{{cookiecutter.project_name}}/docker/django/Dockerfile

```diff
@@ -80,15 +81,14 @@ COPY --chown=web:web ./poetry.lock ./pyproject.toml /code/
 
 # Project initialization:
 # hadolint ignore=SC2046
-RUN echo "$DJANGO_ENV" \
+RUN --mount=type=cache,target=$POETRY_CACHE_DIR \
```

Comment:
Just to be sure: do we need to quote `target="$POETRY_CACHE_DIR"`? Because in bash we need to.

Reply:
I have tried it at my local. And there was no problem.

Reply:
> And there was no problem.

With quoting it? Let's add it in this case :)